desktop/apps/categories3/components/FeaturedGene.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { primary } from '@artsy/reaction-force/dist/assets/fonts'
+
+const propTypes = {
+  title: PropTypes.string,
+  href: PropTypes.string,
+  image: PropTypes.object
+}
+
+const Container = styled.div`
```

Comment:
Super minor, but maybe this should be `GeneContainer` to follow the pattern of children?